packages/lit-html/src/lit-html.ts

```diff
@@ -1432,6 +1432,21 @@ export type {EventPart};
 class EventPart extends AttributePart {
   readonly type = EVENT_PART;
 
+  constructor(
+    element: HTMLElement,
+    name: string,
+    strings: ReadonlyArray<string>,
+    parent: Disconnectable | undefined,
+    options: RenderOptions | undefined
+  ) {
+    super(element, name, strings, parent, options);
+
+    if (strings.length > 2 || strings[0] !== '' || strings[1] !== '') {
+      throw new Error('An event listener in a template has extra content ' +
```

Comment:
I think it's better to give more information to help developer locate the position. Such as `name`/`strings`.

Reply:
Pinning down the location might be a job better suited for lit-analyzer since there's not really a lot of context available. I did try this out and ended up going a bit overboard:

```diff
diff --git a/packages/lit-html/src/lit-html.ts b/packages/lit-html/src/lit-html.ts
index 07edbab8..d1e1e566 100644
--- a/packages/lit-html/src/lit-html.ts
+++ b/packages/lit-html/src/lit-html.ts
@@ -1445,9 +1445,18 @@ class EventPart extends AttributePart {
       DEV_MODE &&
       (strings.length > 2 || strings[0] !== '' || strings[1] !== '')
     ) {
+      const messageParts = [
+        {text: `    <${element.localName} ... @${name}="`},
+        {text: strings[0], isError: true},
+        {text: '${...}'},
+        {text: strings.slice(1).join('${...}'), isError: true},
+        {text: '" ...'},
+      ];
       throw new Error(
-        'An event listener in a template has extra content surrounding the ' +
-          'listener expression.'
+        `An event listener on a <${element.localName}> in a template has ` +
+          'extra content surrounding the listener expression:\n' +
+          messageParts.map(x => x.text).join('') + '\n' +
+          messageParts.map(x => (x.isError ? '^' : ' ').repeat(x.text.length)).join('')
       );
     }
   }
```

which spits out errors like this:

```
Error: An event listener on a <div> in a template has extra content surrounding the listener expression:
    <div ... @click="${...}}" ...
                           ^     
```

when the code looks like this:

```js
render(html`<div @click=${listener}}></div>`, container);
```

But there's no way to give a file and line number or print a more accurate representation of the source with what's available, as far as I can tell.